plugins/admin.sma

```diff
@@ -354,57 +354,49 @@ AddAdmin(id, auth[], accessflags[], password[], flags[], comment[]="")
 
 loadSettings(szFilename[])
 {
-	new File=fopen(szFilename,"r");
+	new File = fopen(szFilename, "r");
 	
 	if (File)
 	{
-		new Text[512];
-		new Flags[32];
-		new Access[32]
-		new AuthData[44];
-		new Password[32];
-		
+		new szString[512], szFlags[32], szAccess[32], szAuth[44], szPassword[32];
 		while (!feof(File))
 		{
-			fgets(File, Text, charsmax(Text));
+			fgets(File, szString, charsmax(szString));
 			
-			trim(Text);
+			trim(szString);
 			
-			// comment
-			if (Text[0]==';') 
+			if(!szString[0] || szString[0] == ';' || szString[0] == '/' && szString[1] == '/')
```

Comment:
if (!szString[0] || szString[0] == ';' || szString[0] == '/' && szString[1] == '/')

You probably mean

if (szString[0] == '^0' || szString[0] == ';' || **(**szString[0] == '/' && szString[1] == '/'**)**)